### PR TITLE
Wip: Implement json based theme

### PR DESCRIPTION
**Pull request by TheOneRing**

### CODE REVIEW

File: theme/config_with_defaults.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "desktop_msi_upgrade_code_text": "",
+    "desktop_application_short_name_text": "ownCloud",
+    "desktop_settings_server_user_check": false,
+    "desktop_change_url_auth_select": "fromconfig",
+    "desktop_settings_oauth2_check": true,
+    "desktop_settings_oauth2_clientid_text": "xdXOt13JKxym1B1QcEncf2XDkLAexMBFwiT9j6EfhhHFJhs2KM9jbjTmf8JBXE69",
+    "desktop_settings_oauth2_secretid_text": "UBntmLjC2yYCeHwsyj73Uwo9TAaecAetRwMw0xYcvNL9yRdLSUi0hUAHfvCHFeFh",
+    "desktop_settings_oidc_scopes_text": "openid offline_access email profile",
+    "desktop_settings_show_help_url_check": true,
+    "desktop_settings_help_url_text": "https://doc.owncloud.org/desktop/latest/",
```
Comment (dschmidt) [inline theme/config_with_defaults.json:17]: Formerly `https://www.owncloud.org/help` was set as `ARPHELPLINK` now it'll be this

File: theme/config_with_defaults.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{
+    "desktop_git_branch_text": "v2.9.1",
+    "desktop_application_name_text": "ownCloud",
+    "desktop_webdav_check": true,
+    "desktop_webdav_url_text": "remote.php/webdav/",
+    "desktop_sync_folder_target_text": "/",
+    "desktop_settings_multiaccount_check": false,
+    "desktop_msi_upgrade_code_text": "",
```
Comment (dschmidt) [inline theme/config_with_defaults.json:8]: `EB15081D-1892-4DB5-872C-9424BFFFF22F`

File: theme/config_with_defaults.json (hunk 1 above)
Comment (TheOneRing) [reply theme/config_with_defaults.json:17]: Maybe we should no longer use org links?

Comment (dschmidt) [reply theme/config_with_defaults.json:17]: No preference I was just documenting effects/changes

**Pull request closed without merging**